packages/source-ghost/index.js

```diff
@@ -72,8 +62,7 @@ class GhostSource {
 
   async loadAuthors ({ addContentType, addCollection = addContentType }) {
```

Comment:
```suggestion
  async loadAuthors ({ addCollection }) {
```
Can probably remove the `addContentType` fallback since this version will only work with Gridsome v0.7 anyway :)